deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -8,12 +8,15 @@ metadata:
           "apiVersion": "infra.watch/v1alpha1",
           "kind": "ServiceAssurance",
           "metadata": {
-            "name": "saf-default"
+            "name": "cloud1"
```

Comment:
We've been using `saf-default` not `cloud1` as the default name for an SAF instance. I think @mrunge might be depending on that name for the smoke tests as well.